default.config.yml

```diff
@@ -42,6 +42,10 @@ vagrant_plugins:
   - name: vagrant-vbguest
   - name: vagrant-hostsupdater
 
+# Automatically fix timeout issues with NFS on macOS. See
+# https://github.com/geerlingguy/drupal-vm/issues/2154
+vagrant_nfs_fix_enabled: "{{ vagrant_synced_folder_default_type == 'nfs' }}"
```

Comment:
Still not sure if this should really run by default just because it's nfs. Maybe it should be opt-in?

Reply:
Since it seems to only be a problem on Big Sur, maybe it should be opt in?

Reply:
Yeah... maybe just set it false and we can add a note in the docs under this page? http://docs.drupalvm.com/en/latest/getting-started/installation-macos/ like:

## Big Sur

If you are using NFS shares on macOS Big Sur, and you have problems with the NFS mount going away, please set `vagrant_nfs_fix_enabled: true` in your `config.yml`.

Something like that.

Reply:
Yep, sounds good!